### PR TITLE
Remove old alipay icon

### PR DESCRIPTION
**Pull request by shopiwags**
There are 2 icons for alipay and one (the old one) is no longer needed.

### CODE REVIEW

**Comment by larouxn** [comment]
Should we rename `alipay_cn` to `alipay` at the same time so the default Alipay icon is simply called `alipay`?

File: db/payment_icons.yml [hunk 1]
```diff
@@ -819,7 +815,7 @@
   label: PostFinance E-Finance
   group: bank_transfers
 -
-  name: alipay_cn
+  name: alipay
   label: AliPay
   group: other
```
Comment (tauthomas01) [inline db/payment_icons.yml:820]: Should it be `wallets` or `other`?

Comment (shopiwags) [reply db/payment_icons.yml:820]: It should be under `other`

**Review by larouxn — APPROVED** [review]
LGTM. We'll need to release as a minor version bump with ⚠️ notes to make it clear to people we've both removed an icon and changed another.

**Review by tauthomas01 — APPROVED** [review]

**Pull request merged**